issue10: Adding simple interface for lines

pthien opened the pull request:
Change default data plotted to consist of 3 data sets - 3 random walks, and a 3D sinc function

a7g4 commented on src/rendering/mod.rs:
```diff
@@ -10,6 +10,13 @@ pub struct Vertex {
 }
+
+pub struct Line {
+    pub indicies: Vec<u32>,
```
You mentioned potentially moving this out of the struct. Can you make an issue for that?

pthien replied:
done https://github.com/dr-sg/rscat/issues/13

a7g4 commented on src/main.rs:
```diff
@@ -142,13 +161,21 @@ fn main() {
+                for i in 0..lines.len() {
+                    let v = &lines[i].verticies;
+                    let i = &lines[i].indicies;
+                    renderer.render(&mut commands, &frame.view, v, &i, false);
```
This is going to render individual points on the line as unconnected dots; is that your intent?

(And if it is, do you want to eventually support the connections? If so, make an issue 😃 )

pthien replied:
done. https://github.com/dr-sg/rscat/issues/12

pthien committed:
issue10: Adding simple interface for lines

Change default data plotted to consist of 3 data sets - 3 random walks, and a 3D sinc function
